shardy/dialect/sdy/ir/attrs.td

```diff
@@ -148,6 +148,18 @@ def Sdy_SubAxisInfo : AttrDef<Sdy_Dialect, "SubAxisInfo"> {
     axis sizes to its left `m=prod(k_1,...,k_(i-1))` (aka pre-size) and size
     k_i. Therefore, the sub-axis-info attribute holds those two numbers and is
     denoted as follows: `(m)k` for pre-size m and size k.
+
+    Constraints:
```

Comment:
Shall we make this bold and remove newline after it, similar to options in https://github.com/openxla/shardy/pull/276/files